docs/build/messages/custom.mdx

```diff
@@ -0,0 +1,623 @@
+---
+sidebar_label: Custom
+sidebar_position: 12
+description: Learn how to build custom content types for use with XMTP
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Build custom content types for use with XMTP
+
+:::caution
+
+Your custom content type WILL NOT automatically be supported by other apps and will display fallback text in them instead.
```

Comment:
> and will display fallback text in them instead

this reads as definite when it's possible that a custom content type is just ignored by a client app.